src/main/java/rip/bolt/ingame/commands/TournamentAdminCommands.java

```diff
@@ -5,21 +5,42 @@
 import org.bukkit.OfflinePlayer;
 import org.bukkit.command.CommandSender;
 
+import org.jvnet.hk2.annotations.Optional;
+import rip.bolt.ingame.Tournament;
 import rip.bolt.ingame.TournamentManager;
+import rip.bolt.ingame.format.TournamentFormat;
 import rip.bolt.ingame.team.ConfigTeamParser;
 import rip.bolt.ingame.team.TournamentPlayer;
 import rip.bolt.ingame.team.TournamentTeam;
 import rip.bolt.ingame.team.TournamentTeamManager;
+import rip.bolt.ingame.utils.UUIDFetcher;
 import rip.bolt.ingame.xml.MapFormatXMLParser;
 import tc.oc.pgm.api.match.Match;
 import tc.oc.pgm.lib.app.ashcon.intake.Command;
 import tc.oc.pgm.lib.app.ashcon.intake.parametric.annotation.Text;
 
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.UUID;
+
 public class TournamentAdminCommands {
 
     @Command(aliases = "create", desc = "Creates a tournament", usage = "<format>", perms = "ingame.staff")
-    public void tourney(CommandSender sender, TournamentManager manager, Match match, @Text String pool) {
-        manager.createTournament(match, MapFormatXMLParser.parse(pool));
+    public void tourney(CommandSender sender, TournamentManager manager, Match match, @Nullable String pool) {
+        if(pool == null){
+            sender.sendMessage(ChatColor.GOLD + "------- " + ChatColor.AQUA + "Loaded formats" + ChatColor.GOLD + " -------");
+            for (String format : MapFormatXMLParser.getFilesFromFolder("formats", ".xml"))
```

Comment:
```suggestion
            for (String format : MapFormatXMLParser.getAvailableFormats())
```
The extension is very redundant when you're calling an explicity MapFormat**XML**Parser, and i'd also say that the command deciding what folder ("formats") to look at doesn't make much sense either, since it's not like parsing will let you pick a different folder.